test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -0,0 +1,173 @@
+Describe "Set/New-Service cmdlet tests" -Tags "Feature", "RequireAdminOnWindows" {
+    BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if ( -not $IsWindows ) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+    }
+    AfterAll {
+        $global:PSDefaultParameterValues = $originalDefaultParameterValues
+    }
+
+    It "SetServiceCommand can be used as API for '<parameter>' with '<value>'" -TestCases @(
+        @{parameter = "ComputerName"; value = "foo"},
+        @{parameter = "Name"        ; value = "bar"},
+        @{parameter = "DisplayName" ; value = "hello"},
+        @{parameter = "Description" ; value = "hello world"},
+        @{parameter = "StartupType" ; value = "Automatic"},
+        @{parameter = "StartupType" ; value = "Boot"},
+        @{parameter = "StartupType" ; value = "Disabled"},
+        @{parameter = "StartupType" ; value = "Manual"},
+        @{parameter = "StartupType" ; value = "System"},
+        @{parameter = "Status"      ; value = "Running"},
+        @{parameter = "Status"      ; value = "Stopped"},
+        @{parameter = "Status"      ; value = "Paused"},
+        @{parameter = "InputObject" ; value = (Get-Service | Select-Object -First 1)},
```

Comment:
It is failed on Unix #4800
We should move `(Get-Service | Select-Object -First 1)` to BeforeAll.